### PR TITLE
4.1 documentation check

### PR DESCRIPTION
**Pull request by rpspringuel**
To prepare for 4.1 I looked over the documentation and found (and fixed) a few problems.

### CODE REVIEW

File: doc/Command_Index_gregorio.tex [hunk 1]
```diff
@@ -751,9 +751,6 @@ \section{Gregorio Controls}
   \#4 & \TeX\ code & Macros rendering the first syllable; should emit the initial and populate \verb=\gre@opening@syllabletext=.\\
 \end{argtable}
 
-\macroname{\textbackslash GreScoreReference}{\#1}{gregoriotex-main.tex}
```
Comment (henryso) [inline doc/Command_Index_gregorio.tex]: This may do nothing, but its use has not been removed.

**Comment by henryso** [comment]
Other than that, it looks OK to me.

File: doc/Command_Index_gregorio.tex (hunk 1 above)
Comment (rpspringuel) [inline doc/Command_Index_gregorio.tex]: The function is marked as obsolete and will raise an error if used.  No other obsolete or deprecated functions are documented.

If it's supposed to be there (even doing nothing), then it shouldn't be raising an obsolete error.

Comment (henryso) [inline doc/Command_Index_gregorio.tex]: Ok, then the mismatch is on the C side.  The C code still generates this.  I'll create an issue to take care of that.

**Pull request merged**